### DISCUSSION TITLE
New styles : MK JSON

### DISCUSSION THREAD

**Discussion by K3nt3L** [lllyasviel/Fooocus#1378] [Show and tell]
You can download my new styles here :
[https://github.com/K3nt3L/MK-Styles](https://github.com/K3nt3L/MK-Styles)

JSON format
Demo here :
[Twitter showcase video](https://twitter.com/kentelTICE/status/1734649439506035151)

**Comment by lllyasviel** [comment]
hi this is added in 2.1.853
thanks for the contribution!

**Reply by K3nt3L** [reply]
Awesome. 
:)
